blog-posts/case-fc-bayern.md

```diff
@@ -0,0 +1,66 @@
+---
+path: "/blog/case-fc-bayern/"
```

Comment:
I think we should optimize the url in terms of SEO. @feedm3  any suggestions?

`fc-bayern-website-frontend-relaunched-by-satellytes` or such ?